kasa/iot/modules/ambientlight.py

```diff
@@ -14,9 +15,27 @@
 class AmbientLight(IotModule):
     """Implements ambient light controls for the motion sensor."""
 
+    def __init__(self, device, module):
+        super().__init__(device, module)
+        self._add_feature(
+            Feature(
+                device=device,
+                container=self,
+                name="Ambient Light",
+                icon="mdi:brightness-percent",
+                attribute_getter="ambientlight_brightness",
+                type=FeatureType.Sensor,
+            )
+        )
+
     def query(self):
         """Request configuration."""
-        return self.query_for_command("get_config")
```

Comment:
Perhaps the `get_config` should still be kept, perhaps just as a regular method in case it is not completely useless?

Reply:
Whoops, that was a copy/paste error.  Swapped it back to get_config.